cmd/thanos/compact.go

```diff
@@ -463,3 +479,7 @@ func generateIndexCacheFile(
 	}
 	return nil
 }
+
+func isEnableDedup(enableDedup bool, dedupReplicaLabel string) bool {
```

Comment:
Consider getting rid of this one line function, IMO no point in abstracting single line of code :)

Reply:
Ok, I can remove this function.